hack/smoke.sh

```diff
@@ -87,11 +87,10 @@ fi
 
 check_command curl
 check_command jq
-check_command mktemp
 
 RESPONSE=$(curl --oauth2-bearer "${TOKEN}" -sSfL "${CLUSTER_URL}/api/k8s/apis/workspaces.konflux-ci.dev/v1alpha1/workspaces")
 ARGS=".items[] | select(.metadata.namespace == \"${USERNAME}\")"
-OUTPUT="$(jq "${ARGS}" "${RESPONSE}")"
+OUTPUT="$(echo "${RESPONSE}" | jq "${ARGS}")"
```

Comment:
If you prefer we can use redirection

```suggestion
OUTPUT=$(jq "${ARGS}" <<< "${RESPONSE}")
```

Reply:
lgtm